viewer/test/viewer-test-pptr.js

```diff
@@ -216,7 +216,7 @@ describe('Lighthouse Viewer', () => {
           document.querySelector('.lh-tools__dropdown')).visibility === 'visible';
       });
 
-      // For some reason, clicking this button doesn't always initiate the download after upgrading to Puppeteer 15.2.0.
+      // For some reason, clicking this button doesn't always initiate the download after upgrading to Puppeteer 16.
```

Comment:
why update this line? presumably the version it broke in is useful information.

Reply:
I didn't do a bisect to find the revision that broke it, this is just calling out the version upgrade that introduced the problem for us.